samples/mentions.html

```diff
@@ -0,0 +1,271 @@
+								</p>
+							</div>
+						</li>
+						<li>
+							<img src="assets/mentions/img/w_1.jpg" alt="avatar" class="photo" />
+							<div class="message">
+								<strong>Mildred Wilson</strong>
+								<span class="bullet">•</span>
+								<a class="mailto-user" href="mailto:mwilson@example.com">@mwilson</a>
+								<span class="bullet">•</span>
+								<span class="time">4 hours ago</span>
+								<p class="post-content">
+								Rating has been discontinued because it caused a decline in content quality rather than its increase. People were providing content that would be upvoted, not the content valuable for the community. There are even “secret” groups now!
+								</p>
+							</div>
+						</li>
+					</ul>
+					<div class="editor">
+						<textarea cols="80" id="editor1" name="editor1" rows="10" data-sample="1" data-sample-short>
+							I agree with <a href="mailto: mildred@example.com">@mwilson</a>. Rating system made the quality much worse. In the long run, especially when we have more new members, it will be too hard to control.
+						</textarea>
+					</div>
+					<button id="send" class="send">Send</button>
+				</div>
+
+			</section>
+		</section>
+
+		<footer class="sdk-footer">
+		</footer>
+
+		<script data-sample="1" >
+			var users = [
+				{ id: 1, avatar: 'm_1', fullname: 'Charles Flores' },
+				{ id: 2, avatar: 'm_2', fullname: 'Gerald Jackson' },
+				{ id: 3, avatar: 'm_3', fullname: 'Wayne Reed' },
+				{ id: 4, avatar: 'm_4', fullname: 'Louis Garcia' },
+				{ id: 5, avatar: 'm_5', fullname: 'Roy Wilson' },
+				{ id: 6, avatar: 'm_6', fullname: 'Matthew Nelson' },
+				{ id: 7, avatar: 'm_7', fullname: 'Randy Williams' },
+				{ id: 8, avatar: 'm_8', fullname: 'Albert Johnson' },
+				{ id: 9, avatar: 'm_9', fullname: 'Steve Roberts' },
+				{ id: 10, avatar: 'm_10', fullname: 'Kevin Evans' },
+
+				{ id: 11, avatar: 'w_1', fullname: 'Mildred Wilson' },
+				{ id: 12, avatar: 'w_2', fullname: 'Melissa Nelson' },
+				{ id: 13, avatar: 'w_3', fullname: 'Kathleen Allen' },
+				{ id: 14, avatar: 'w_4', fullname: 'Mary Young' },
+				{ id: 15, avatar: 'w_5', fullname: 'Ashley Rogers' },
+				{ id: 16, avatar: 'w_6', fullname: 'Debra Griffin' },
+				{ id: 17, avatar: 'w_7', fullname: 'Denise Williams' },
+				{ id: 18, avatar: 'w_8', fullname: 'Amy James' },
+				{ id: 19, avatar: 'w_9', fullname: 'Ruby Anderson' },
+				{ id: 20, avatar: 'w_10', fullname: 'Wanda Lee' }
+			],
+
+			buttonEl = document.getElementById( 'send' );
+
+		// Lets create some fake usernames from fullnames.
+		CKEDITOR.tools.array.forEach( users, function( user ) {
+			var username = user.fullname.split( ' ' );
+			username[ 0 ] = username[ 0 ].substring( 0, 1 );
+			user.username = username.join( '' ).toLowerCase();
+		} );
+
+		CKEDITOR.replace( 'editor1', {
+			extraPlugins: 'mentions,emoji',
+			removeDialogTabs: 'link:advanced',
```

Comment:
Instead of getting rid of plugins, it could be better to simply whitelist allowed plugins using `config.plugins` properties.

Just provide bare minimum of plugins, basically those that you now use in the toolbar.

As a result you'll be able to remove even more properties, like `removeDialogTabs` or `resize_enabled`.